hle58-01/petri_models/ice_tea/ice_tea_model.py

```diff
@@ -0,0 +1,266 @@
+
+from helpers import (
+    messageHandler, calculateChangeHandler,
+    numberConverterHandler, tokenHandler,
+    remainingValueHandler
+)
+
+# Global variables
+transitionStatus = str()
+
+# Define the Petri net transitions for icetea
+def iced_tea_transitions(
+        tokens, iced_teaTransitions, buttons, coins, drink_var, coin_var, currentValue,
+        productsList, canvas, messages_text,
+):
+    global transitionStatus
+
+    def transition0():
+        global transitionStatus
+        if drink_var.get() != "null" and iced_teaTransitions["canRunTransition0"]:
+            iced_teaTransitions["prevTransition0"] = True
+            transitionStatus = "T0"
+            return True
+        else:
+            return False
+        
+    def transition1():
+        global transitionStatus
+        if currentValue >= productsList["icetea"]["value"] and iced_teaTransitions["prevTransition0"]:
+            iced_teaTransitions["prevTransition0"] = False
+            iced_teaTransitions["prevTransition1"] = True
+            transitionStatus = "T1"
+            return True
+        else:
+            return False
+
+    def transition15():
```

Comment:
is there a reason why transition15 and 16 are not placed after transition 14?
if not you can change it for concistency.